BostonData/Intents/get_alerts.py

```diff
@@ -0,0 +1,32 @@
+from bs4 import BeautifulSoup
```

Comment:
We need to pull this library into the lamba zip file by editing `requirements.txt` (see comment on this PR for instructions).